core/src/main/scala/org/apache/spark/rpc/RpcEndpoint.scala

```diff
@@ -33,9 +33,9 @@ private[spark] trait RpcEnvFactory {
  *
  * It is guaranteed that `onStart`, `receive` and `onStop` will be called in sequence.
  *
- * The life-cycle of an endpoint is:
+ * The life-cycle of an endpoint is as below in an order:
```

Comment:
Can we just wrap this block as code? The rewording is confusing and doesn't read as clearly to me.